### PR TITLE
select gas token of chain as `from` token by default

### PR DESCRIPTION
**Pull request by mintdart**

### CODE REVIEW

File: src/hooks/useQueryParams.tsx [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+	const { chain: chainOnWallet } = useNetwork();
 
 	const { chain: chainOnURL, from: fromToken, to: toToken } = router.query;
 
 	const chainName = typeof chainOnURL === 'string' ? chainOnURL.toLowerCase() : 'ethereum';
 	const fromTokenAddress = typeof fromToken === 'string' ? fromToken.toLowerCase() : null;
 	const toTokenAddress = typeof toToken === 'string' ? toToken.toLowerCase() : null;
 
+	useEffect(() => {
+		if (router.isReady && isConnected && !chainOnURL && chainOnWallet) {
```
Comment (0xngmi) [inline src/hooks/useQueryParams.tsx:21]: what does `router.isReady` do?

**Review by 0xngmi — APPROVED** [review]

**Comment by 0xngmi** [comment]
when i go to https://llamaswap-git-gastoken-default-select-llamapay.vercel.app/ ETH should be selected as from token too

File: src/hooks/useQueryParams.tsx (hunk 1 above)
Comment (mintdart) [reply src/hooks/useQueryParams.tsx:21]: on first render, even tho there are query params in url, chainOnUrl, fromToken, toToken are undefined, `router.isReady`  is a check to make sure we don't redirect unnecessarily when there is a valid token address in url

**Comment by mintdart** [comment]
> when i go to https://llamaswap-git-gastoken-default-select-llamapay.vercel.app/ with no wallet connected ETH should be selected as from token too

okay, I updated it

**Review by vrtnd — APPROVED** [review]

**Pull request merged**